airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -272,6 +276,8 @@ def __init__(
         pod_runtime_info_envs: list[k8s.V1EnvVar] | None = None,
         termination_grace_period: int | None = None,
         configmaps: list[str] | None = None,
+        deferrable: bool = False,
+        poll_interval: float = 2,
```

Comment:
What does `poll_interval` mean for non-async operator?

Reply:
For non-async nothing but it will give user opportunity to configure how often trigger should check pod operator status if deferrable mode is on

Reply:
Please update docstring accordingly on L212